ui/selectors/selectors.js

```diff
@@ -110,7 +110,7 @@ export function isHardwareWallet(state) {
  */
 export function getHardwareWalletType(state) {
   const keyring = getCurrentKeyring(state);
-  return keyring.type.includes('Hardware') ? keyring.type : undefined;
+  return keyring?.type?.includes('Hardware') ? keyring.type : undefined;
```

Comment:
[nit] this could be `isHardwareWallet(state) ? keyring.type : undefined`

Reply:
@adonesky1 definitely could, fixed!